### PR TITLE
fix: avoid to array mutation

### PR DESCRIPTION
**Pull request by wozny1989**
# Task
 - Avoid to extend prototypes passed array by `EmberArray` - there is a problem on Ember app with set `EXTEND_PROTOTYPES: false,` in `environment.js`
 - fixes: https://github.com/NullVoxPopuli/ember-composable-helpers/issues/18

### CODE REVIEW

File: ember-composable-helpers/src/helpers/next.js [hunk 1]
```diff
@@ -14,7 +13,7 @@ export function next(currentValue, maybeArray, useDeepEqual = false) {
     return;
   }
 
-  return currentIndex === lastIndex ? currentValue : emberArray(array).objectAt(currentIndex + 1);
+  return currentIndex === lastIndex ? currentValue : array.at(currentIndex + 1);
```
Comment (NullVoxPopuli) [inline ember-composable-helpers/src/helpers/next.js:16]: very nice!

File: test-app/tests/integration/helpers/includes-test.js [hunk 2]
```diff
@@ -50,8 +50,14 @@ module('Integration | Helper | {{includes}}', function(hooks) {
     await render(hbs`{{includes this.selectedGame this.wishlist}}`);
 
     assert.dom().hasText('true', 'should render true');
-
-    run(() => this.get('wishlist').removeObject(games[0]));
+    assert.true(!this.wishlist.removeObject, 'should not extend the array with additional functions')
```
Comment (NullVoxPopuli) [inline test-app/tests/integration/helpers/includes-test.js:53]: thanks for this!

**Review by NullVoxPopuli — APPROVED** [review]

**Comment by NullVoxPopuli** [comment]
looks like tests need an update!

once that's fixed I can merge and release.

sorry for not getting to this sooner -- it turns out that I wasn't subscribed to this repo!

**Comment by wozny1989** [comment]
@NullVoxPopuli I just resolved conflicts, let's check CI 🤞

**Review by NullVoxPopuli — APPROVED** [review]

**Pull request merged**